include/FootPrintsInterpolator.h

```diff
@@ -31,6 +32,12 @@ typedef struct{
     double initialAcceleration;
 } InitialState;
 
+typedef struct{
+    iDynTree::Vector2 initialPosition;
+    iDynTree::Vector2 initialVelocity;
+} DcmInitialState;
```

Comment:
Maybe we can discuss on how to extend the ``InitialState`` struct to satisfy both needs, maybe by using a variable size vector instead of doubles.